.github/workflows/ci.yml

```diff
@@ -665,12 +665,23 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{ secrets.TRINO_AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.TRINO_AWS_SECRET_ACCESS_KEY }}
           AWS_REGION: ${{ vars.TRINO_AWS_REGION }}
-          S3_BUCKET: ${{ vars.TRINO_S3_S3FS_TESTS_BUCKET }}
         if: >-
           contains(matrix.modules, 'trino-filesystem-s3') && contains(matrix.profile, 'cloud-tests') &&
           (env.CI_SKIP_SECRETS_PRESENCE_CHECKS != '' || env.AWS_ACCESS_KEY_ID != '' || env.AWS_SECRET_ACCESS_KEY != '')
         run: |
+          # Create an empty S3 bucket for S3 filesystem cloud tests and add the bucket name to GitHub environment variables
+          .github/bin/s3/setup-empty-s3-bucket.sh
+          echo "EMPTY_S3_BUCKET=$(cat .github/bin/s3/.bucket-identifier)" >> $GITHUB_ENV
```

Comment:
why write to GITHUB_ENV?

also, i'd expect `export` here, i think it won't work without it

Reply:
It didn't work before when I tried to source the script so I figured it wouldn't work if I did that

Reply:
will `$GITHUB_ENV` be effective starting from next step?
and we need this on the very next line of the shell step?

> It didn't work before when I tried to source the script 

what exactly did not work?

Reply:
The check failed since the test wasn't able to find the env var

Reply:
@nineinchnick are you able to explain this to me? otherwise it's mind-blowing.

Reply:
Actually I don't think this will work. I just read the docs here https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable

Reply:
So now I'm confused. How do I set an environment variable that will be visible from the tests? It didn't work by sourcing a script that exports the environment variable before

Reply:
I could use `$GITHUB_ENV` if I set `EMPTY_S3_BUCKET` in a preceding step to the testing step

Reply:
can you use `export` here pleasE?

Reply:
Absolutely, changed to export

Reply:
Could you run test-with-secrets so we can check if that works?